apps/web/src/pages/templates/workflow/workflow/node-types/ChannelNode.tsx

```diff
@@ -53,6 +58,30 @@ export default memo(
 
       setCount(foundIndex + 1);
     }, [nodes.length, data, id]);
+    const subtitle = useMemo(() => {
+      const content = data.content;
+      if (StepTypeEnum.DELAY === data.channelType) {
+        return delaySubtitle(data);
+      }
+      if (StepTypeEnum.DIGEST === data.channelType) {
+        return <WillBeSentHeader index={data.index} isHighlight={false} />;
+      }
+
+      if (typeof content === 'string') {
+        return content;
+      }
+
+      if (data.channelType === StepTypeEnum.EMAIL) {
+        if (content && content?.length > 0) {
+          return content?.find((item) => item.type === 'text')?.content;
```

Comment:
currently, it will show the text content of type text, button text will be ignored
not sure if we should show the button text as well